specs/core/0_beacon-chain.md

```diff
@@ -812,7 +813,8 @@ def get_active_index_root(state: BeaconState,
     """
     Return the index root at a recent ``epoch``.
     """
-    assert get_current_epoch(state) - LATEST_ACTIVE_INDEX_ROOTS_LENGTH + ACTIVATION_EXIT_DELAY < epoch <= get_current_epoch(state) + ACTIVATION_EXIT_DELAY
+    min_epoch = get_current_epoch(state) + ACTIVATION_EXIT_DELAY - LATEST_ACTIVE_INDEX_ROOTS_LENGTH + 1 if epoch > LATEST_ACTIVE_INDEX_ROOTS_LENGTH - ACTIVATION_EXIT_DELAY else GENESIS_EPOCH
```

Comment:
1. I think you wanted to use both `get_current_epoch(state)` here? `... if get_current_epoch(state) > LATEST_ACTIVE_INDEX_ROOTS_LENGTH - ACTIVATION_EXIT_DELAY ...`
2. Though I can see `LATEST_ACTIVE_INDEX_ROOTS_LENGTH` definitely is greater than `ACTIVATION_EXIT_DELAY`, but it added a config validity condition https://github.com/ethereum/eth2.0-specs/issues/407.


Solution: keep it nonnegative
```suggestion
    min_epoch = get_current_epoch(state) + ACTIVATION_EXIT_DELAY - LATEST_ACTIVE_INDEX_ROOTS_LENGTH + 1 if get_current_epoch(state) + ACTIVATION_EXIT_DELAY > LATEST_ACTIVE_INDEX_ROOTS_LENGTH else GENESIS_EPOCH
```